lib/std/os/windows.zig

```diff
@@ -171,8 +188,8 @@ pub const WaitForSingleObjectError = error{
     Unexpected,
 };
 
-pub fn WaitForSingleObject(handle: HANDLE, milliseconds: DWORD) WaitForSingleObjectError!void {
-    switch (kernel32.WaitForSingleObject(handle, milliseconds)) {
+pub fn WaitForSingleObject(handle: HANDLE, milliseconds: DWORD, alertable: bool) WaitForSingleObjectError!void {
```

Comment:
This looks like it should now be called `WaitForSingleObjectEx`. Note also that we are transitioning to use NtWaitForSingleObject rather than kernel32 (#1840)